danuluma/dannstore
# #161330034 Handle errors and correct returned status codes

#### What does this PR do?
Implement feedback from LFA

#### Description of Task to be completed?
Handle error 500 and 404
Correct returned status codes on POST requests

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Use postman to send any request to a non-existent endpoint eg http://localhost:5000/dann/api/v1/not-available 
* You should receive the following response ```{
    "Error": "Resource not found"
}```

#### What are the relevant pivotal tracker stories?
[#161330034](https://www.pivotaltracker.com/story/show/161330034)
